src/OpenTK.NT/Native/User32/Structs/RawInputDeviceList.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Runtime.InteropServices;
+
+using HANDLE = System.IntPtr;
+
+namespace OpenTK.NT.Native
+{
+    /// <summary>
+    /// Contains information about a raw input device.
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    public struct RawInputDeviceList
+    {
+        /// <summary>
+        /// Handle to the raw input device.
+        /// </summary>
+        public HANDLE Device;
+
+        /// <summary>
+        /// Type of device.
+        /// </summary>
+        public RawInputDeviceType Type;
+
+        public static readonly uint SizeInBytes = (uint)Marshal.SizeOf<RawInputDeviceList>();
```

Comment:
Required?

Reply:
It's a parameter of `GetRawInputDeviceList`